arch/arm64/exceptions_c.c

```diff
@@ -18,6 +18,123 @@ struct fault_handler_table_entry {
     uint64_t fault_handler;
 };
 
+struct fault_status_map {
+    uint32_t fsc;
+    const char *fault_msg;
+};
+
+/* Instruction and Data abort share the fault status encoding */
+static struct fault_status_map fsc_map[] = {
```

Comment:
should be able to mark this static const so it can live in the rodata section. compiler might be smart enough to figure that out based on usage, but probably better to be explicit.

Reply:
Ack.